### PR TITLE
Merge stable to master

### PR DESCRIPTION
**Pull request by MikaelSmith**

### CODE REVIEW

**Comment by MikaelSmith** [comment]
@underscorgan

File: configs/components/puppet-ca-bundle.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "url": "git://github.com/puppetlabs/puppet-ca-bundle.git",
+  "url": "git@github.com:puppetlabs/puppet-ca-bundle.git",
```
Comment (MikaelSmith) [inline configs/components/puppet-ca-bundle.json]: It is public. I'm not sure about ssh keys. The URLs are inconsistent; currently pxp-agent and cpp-pcp-client are the only others using `git@`.

Comment (joshcooper) [inline configs/components/puppet-ca-bundle.json]: Seems like we should fix those URLs in stable (in a separate PR) and merge that up to master, so we don't get different behavior across branches. /cc @melissa @underscorgan 

:+1: to merge this PR as-is

**Comment by MikaelSmith** [comment]
This would be really nice to merge this, but it's unrelated to any releases.

**Comment by melissa** [comment]
I believe this might need to be updated, as the security stuff was merged into stable yesterday. If that's not the case, and this already includes the cve changes, then I'm :+1:

**Comment by MikaelSmith** [comment]
Ok, I'll remerge.

**Comment by MikaelSmith** [comment]
Updated.

**Comment by melissa** [comment]
:+1:  I'm fine with this getting merged. We should standardize the git urls, but that's for a later PR

File: configs/components/puppet-ca-bundle.json (hunk 1 above)
Comment (melissa) [inline configs/components/puppet-ca-bundle.json]: https://github.com/puppetlabs/puppet-agent/commit/e0b4ba1b9f1db5c4caa52503d82d22a2b95d0a23 this was an intentional change made in puppet-agent#master. We should try to keep it around...

Comment (MikaelSmith) [inline configs/components/puppet-ca-bundle.json]: Should I cherry-pick that up to stable?

Comment (melissa) [inline configs/components/puppet-ca-bundle.json]: I don't know what the best answer is... I guess cherry-picking them to stable would be the cleanest thing to do?

Comment (MikaelSmith) [inline configs/components/puppet-ca-bundle.json]: I'll just avoid merging for now.

File: bin/build-windows.rb [hunk 2]
```diff
@@ -119,14 +124,14 @@ def clone_and_rynsc_private_repo(fork, ref, hostname, ssh_key)
 puts "Build-Windows.rb... building cpp-pcp-client"
 Kernel.system("set -vx;#{scp_command} #{File.join(SCRIPT_ROOT, 'build-cpp-pcp-client.ps1')} Administrator@#{hostname}:/home/Administrator/")
 fail "Copying build-cpp-pcp-client.ps1 to #{hostname} failed" unless $?.success?
-clone_and_rynsc_private_repo(CPPPCPCLIENT['url'], CPPPCPCLIENT['ref'], hostname, ssh_key)
+clone_and_rynsc_private_repo(CPPPCPCLIENT['url'], CPPPCPCLIENT['ref'], hostname, ssh_key, 'cpp-pcp-client')
```
Comment (MikaelSmith) [inline bin/build-windows.rb]: Do we want to preserve this?

**Comment by MikaelSmith** [comment]
Dropped the `git@` change.

File: bin/build-windows.rb (hunk 2 above)
Comment (melissa) [inline bin/build-windows.rb]: Yes, this was a change that was made in the cve branch. It ultimately should be in both master and stable

**Comment by melissa** [comment]
Okie dokie, I think we're good to go on merging this!

File: configs/components/puppet-ca-bundle.json (hunk 1 above)
Comment (MikaelSmith) [inline configs/components/puppet-ca-bundle.json]: Cherry-pick was messy, PR at https://github.com/puppetlabs/puppet-agent/pull/574.

**Comment by joshcooper** [comment]
:+1:

**Pull request merged**